### PR TITLE
doc: Adding first draft of CONTRIBUTING.md and modifying README to reference it.

### PR DESCRIPTION
**Pull request by DjangoFett**
.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Contributing
+First of all, many thanks for considering making a contribution to this project.
+
+# Submitting PRs
+Please do not include any changes to `package-lock.json` in your PRs, even if you are updating package.json. These changes are almost impossible to review for security implications. If necessary, `package-lock.json` will be regenerated and committed by a maintainer after your PR is merged.
```
Comment (MikeRalphson) [inline CONTRIBUTING.md:5]:
```suggestion
Please do not include any changes to `package-lock.json` in your PRs, even if you are updating `package.json`. These changes are almost impossible to review for security implications. If necessary, `package-lock.json` will be regenerated and committed by a maintainer after your PR is merged.
```

**Review by MikeRalphson — CHANGES_REQUESTED** [review]
Thanks - looks good. Can you let me know if you want to tackle any of the other items from #144 in this PR, or I can merge and we can do them later.

File: CONTRIBUTING.md (hunk 1 above)
Comment (DjangoFett) [reply CONTRIBUTING.md:5]: We can merge now, and I'll submit another PR with the changes

**Comment by MikeRalphson** [comment]
Thanks!

**Pull request merged**